src/schema/generic.json

```diff
@@ -726,6 +726,11 @@
             "category": "client",
             "description": "Unable to process payment as card details are invalid"
         },
+        {
+            "code": "CvvInvalid",
```

Comment:
Can `CardDetailsInvalid ` cover this?

Reply:
Yes, they can! Website error messages are usually generic, however, they are sometimes specific. Should we return specific errors when possible? Or should we return more generic errors and wait for the second phase of the errors project?

Reply:
I think we need @emmasalih @mprussell here!

Reply:
discussed it with emma, let's not add this error and use `CardDetailsInvalid ` for now for the consistency. (we have the error code for the `type` of the protocol. cvv is under the type)